rs-matter/src/tlv/parser.rs

```diff
@@ -33,6 +33,64 @@ impl<'a> TLVList<'a> {
     }
 }
 
+impl<'a> Display for TLVList<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let tlvlist = self;
+
+        const MAX_DEPTH: usize = 9;
+        const SPACE_BUF: &str = "                                ";
+
+        let space: [&str; MAX_DEPTH] = [
```

Comment:
If this is for space intend, why not something like `writeln!("{:1$}", " ", 4*index)` to get a 4-space indented thing?

Reply:
I just moved this old code from `Packet` (which is no more) to `TLVList` (where I really think it belongs and it needed a new home anyway).